specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2021-12-01/Marketplace.json

```diff
@@ -0,0 +1,2894 @@
+          "description": "Subscriptions ids list"
+        }
+      }
+    },
+    "CollectionsToSubscriptionsMappingProperties": {
+      "description": "The subscriptions list to get the related collections",
+      "type": "object",
+      "properties": {
+        "subscriptionIds": {
+          "description": "Subscriptions ids list",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "CollectionsToSubscriptionsMappingResponse": {
+      "description": "A map of collections subscriptions details",
+      "type": "object",
+      "properties": {
+        "details": {
+          "$ref": "#/definitions/CollectionsToSubscriptionsMappingResponseProperties",
+          "description": "The map of collections subscriptions"
+        }
+      }
+    },
+    "CollectionsToSubscriptionsMappingResponseProperties": {
+      "description": "Collections - subscriptions mapping details, map key is collectionId. In case subscriptions list is null or empty: no subscription from the list is related to that collection.",
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/collectionsSubscriptionsMappingDetails"
+      }
+    },
+    "collectionsSubscriptionsMappingDetails": {
+      "description": "Collection name and related subscriptions list",
+      "type": "object",
+      "properties": {
+        "collectionName": {
+          "description": "Collection name",
+          "type": "string"
+        },
+        "subscriptions": {
+          "description": "Subscriptions ids list",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "StopSellOffersPlansNotificationsList": {
+      "description": "List of stop sell offers and plans notifications.",
+      "type": "object",
+      "properties": {
+        "stopSellNotifications": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StopSellOffersPlansNotificationsListProperties"
+          }
+        }
+      }
+    },
+    "StopSellOffersPlansNotificationsListProperties": {
+      "description": "List of stop sell offers and plans notifications.",
+      "type": "object",
+      "properties": {
+        "offerId": {
+          "description": "Gets offer id",
+          "type": "string",
+          "readOnly": true
+        },
+        "displayName": {
+          "description": "Gets offer display name",
+          "type": "string",
+          "readOnly": true
+        },
+        "isEntire": {
+          "description": "Gets a value indicating whether entire offer is in stop sell or only few of its plans",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "messageCode": {
+          "description": "Gets the notification message id",
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true
+        },
+        "icon": {
```

Comment:
>icon [](http://example.com/codeflow?start=9&length=4)

iconUrl?

Reply:
same here, we are already using icon on previous version and if it is not necessary we'd like to stay with this.
(you can take a look at the previous API version - 2021-06-01), we have it there